docs/calico-with-docker/without-docker-networking/ManualSetup.md

```diff
@@ -28,7 +28,13 @@ They must have the following software installed:
 
 - Docker v1.6 or greater: [Docker][docker]
 - etcd installed and available on each node: [etcd documentation][etcd]
+
+[NJ] What does 'available' mean?  Does this mean that I need a configured and active etcd system, such that I can do 'etcdctl ls /' on each node?  Any recommendations or restrictions on how many etcd masters, or where I can use proxies?
+
```

Comment:
At this stage, we can support most (all?) etcd topologies, with or without proxies.  If you're running a local proxy then we work without config.  If not, you need to configure the correct URL(s) for us to connect to.